app/_locales/zh_TW/messages.json

```diff
@@ -162,7 +162,7 @@
     "description": "helper for inputting hex as decimal input"
   },
   "blockExplorerUrl": {
-    "message": "封鎖 Explorer"
+    "message": "區塊鏈瀏覽器"
```

Comment:
better to translate as `區塊鏈瀏覽器 URL` (to align with following translations)

Reply:
@gasolin  You can review the EN locale file in `app/_locales/en/message.json`, the original message is `Block Explorer` instead of `Block Explorer Url`, so my translation may be much appropriate.